PlantDiary2022FS001/Pages/Index.cshtml.cs

```diff
@@ -19,6 +20,16 @@ public IndexModel(ILogger<IndexModel> logger)
         }
 
         public void OnGet()
+        {
+            GenerateBrand();
```

Comment:
I refactored the BRand logic to its own method, because methods should only do one thing, and GetData was doing multiple things.

Reply:
Another good way to reduce technical debt.